common/cards/alter-egos/hermits/goatfather-rare.ts

```diff
@@ -36,20 +36,25 @@ class GoatfatherRareHermitCard extends HermitCard {
 		pos: CardPosModel,
 		hermitAttackType: HermitAttackType
 	) {
-		const attacks = super.getAttacks(game, instance, pos, hermitAttackType)
+		const attack = super.getAttacks(game, instance, pos, hermitAttackType)
+
+		if (!attack) return
 
 		const {player, opponentPlayer, row, rowIndex} = pos
 
-		if (attacks[0].type !== 'secondary' || !row?.hermitCard) return attacks
+		if (attack.type !== 'secondary' || !row?.hermitCard) return
 
 		const coinFlip = flipCoin(player, row.hermitCard)
 
-		if (coinFlip[0] === 'tails') return attacks
+		if (coinFlip[0] === 'tails') return
```

Comment:
Goatfather should return `attack` when using primary or Anvil Drop flips tails, this prevents those attacks from executing

Reply:
Hmm, fixed this but now i'm wondering. Would it be possible to make it so getAttacks can't return null? Would prevent this type of mistake from happening. There's probably an edge case I'm not thinking of, though

Reply:
Maybe have `getAttacks` return `AttackModel | null` so it must explicitly return a value, even if it not an attack